AppDB/appscale/datastore/cassandra_env/large_batch.py

```diff
@@ -1,13 +1,23 @@
+import sys
 import uuid
 
 from cassandra.query import ConsistencyLevel
 from cassandra.query import SimpleStatement
+from tornado import gen
 from .retry_policies import (BASIC_RETRIES,
                              NO_RETRIES)
+from .utils import deletions_for_entity
```

Comment:
Imho, it's harder to navigate with relative imports. I often open files without noticing its location (from search window or by 'show definition' function of pycharm) so it's harder to understand, what exactly is imported.

According to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) relative imports are __acceptable__ when absolute are __recommended__:

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved 

> However, explicit relative imports are an acceptable alternative to absolute imports

This comment shouldn't block PR, and I suspect that new relative imports are added just to keep style in the file, but anyway I'd like to hear your opinion on this.

Reply:
I have no problem with absolute imports. Previously, I thought that it was rather verbose for namespace-style packages, but I see now that it can make things clearer.